### DISCUSSION TITLE
Does the blazor web app's static rendering mode work?

### DISCUSSION THREAD

**Discussion by Seanxwy** [vikramlearning/blazorbootstrap#397] [Q&A]
Does the blazor web app's static rendering mode work???

**Comment by gvreddy04** [comment]
@Seanxwy 

![image](https://github.com/vikramlearning/blazorbootstrap/assets/2337067/d528e22f-c5ed-48b3-9435-f551d6ae9c91)

Link: https://learn.microsoft.com/en-us/aspnet/core/blazor/components/render-modes?view=aspnetcore-8.0#render-modes
